kikit/export.py

```diff
@@ -71,62 +92,62 @@ def gerberImpl(boardfile, outputdir, plot_plan=fullGerberPlotPlan, drilling=True
 
     board = LoadBoard(boardfile)
 
-    pctl = PLOT_CONTROLLER(board)
-    popt = pctl.GetPlotOptions()
-
-    popt.SetOutputDirectory(plotDir)
-
-    popt.SetPlotFrameRef(False)
-    popt.SetSketchPadLineWidth(FromMM(0.35))
-    popt.SetAutoScale(False)
-    popt.SetScale(1)
-    popt.SetMirror(False)
-    popt.SetUseGerberAttributes(False)
-    popt.SetIncludeGerberNetlistInfo(True)
-    popt.SetCreateGerberJobFile(True)
-    popt.SetUseGerberProtelExtensions(settings["UseGerberProtelExtensions"])
-    popt.SetExcludeEdgeLayer(settings["ExcludeEdgeLayer"])
-    popt.SetScale(1)
-    popt.SetUseAuxOrigin(settings["UseAuxOrigin"])
-    popt.SetUseGerberX2format(False)
-    popt.SetDrillMarksType(0) # NO_DRILL_SHAPE
+    plot_controller = PLOT_CONTROLLER(board)
+    plot_options = plot_controller.GetPlotOptions()
+
+    plot_options.SetOutputDirectory(plotDir)
+
+    plot_options.SetPlotFrameRef(False)
+    plot_options.SetSketchPadLineWidth(FromMM(0.35))
+    plot_options.SetAutoScale(False)
+    plot_options.SetScale(1)
+    plot_options.SetMirror(False)
+    plot_options.SetUseGerberAttributes(False)
+    plot_options.SetIncludeGerberNetlistInfo(True)
+    plot_options.SetCreateGerberJobFile(True)
+    plot_options.SetUseGerberProtelExtensions(settings["UseGerberProtelExtensions"])
+    plot_options.SetExcludeEdgeLayer(settings["ExcludeEdgeLayer"])
+    plot_options.SetScale(1)
+    plot_options.SetUseAuxOrigin(settings["UseAuxOrigin"])
+    plot_options.SetUseGerberX2format(False)
+    plot_options.SetDrillMarksType(0) # NO_DRILL_SHAPE
```

Comment:
I am not pleased about combining renaming and introducing new functionality in a single commit. It makes history unusable. If you want to trace a bug in history, you cannot easily tell what a behavior change was and what was only renaming.

If renaming is desirable, I prefer first to introduce a commit that changes the naming and then implement the new functionality (or in reverse order).

Note that if there is complete refactor of the function, that internally changes how the function operates then I consider it fine to rename and implement at the same time (as there is no meaningful mapping between the old and new function body).

However, I don't require you to change it; I will accept this PR it.

Reply:
Agreed, renaming should've been a separate commit, I'll change it